types/chrome/index.d.ts

```diff
@@ -5698,7 +5711,7 @@ declare namespace chrome.pageAction {
          * Optional.
          * Either a relative image path or a dictionary {size -> relative image path} pointing to icon to be set. If the icon is specified as a dictionary, the actual image to be used is chosen depending on screen's pixel density. If the number of image pixels that fit into one screen space unit equals scale, then image with size scale * 19 will be selected. Initially only scales 1 and 2 will be supported. At least one image must be specified. Note that 'details.path = foo' is equivalent to 'details.imageData = {'19': foo}'
          */
-        path?: any;
+        path?: string | { [index: number]: string };
```

Comment:
Since the `exactOptionalPropertyTypes` typescript flag is set, undefined should be explicitly declared.
```suggestion
        path?: string | { [index: number]: string } | undefined;
```